### PR TITLE
GetterProp - Add type_ann

### PR DESCRIPTION
**Pull request by dsherret**
Adds parsing of a type annotation to `GetterProp`.

This is valid typescript:

```ts
const obj = {
    get test() : string {
        return "";
    }
};
```

https://www.typescriptlang.org/play/?ssl=5&ssc=3&pln=1&pc=1#code/MYewdgzgLgBCBGArGBeGBvAsAKBnmA5gKaxRHQAUAlDAFwzQBOAlmARjvlzIyQK6MwMAETCA3J3wBfHFLFA

### CODE REVIEW

File: ecmascript/parser/tests/typescript/object/getter-prop/input.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const obj = {
+    get prop1(): string {
+        return "";
+    },
+    get prop2() {
+        return 5;
+    }
+};
```
Comment (dsherret) [inline ecmascript/parser/tests/typescript/object/getter-prop/input.ts]: I should add a newline here.

Comment (dsherret) [reply ecmascript/parser/tests/typescript/object/getter-prop/input.ts]: Done.

**Comment by kdy1** [comment]
The pr seems good to me, but I expect swc_ecma_transforms would be broken.  

Can you give me a permission to push to your fork? I'm asking this as I don't like breaking master branch. (Although tracis ci fails, I always check that all tests passes locally)

**Comment by dsherret** [comment]
@kdy1 that would be great! I believe you already have permission. to push to my fork. Thanks and no rush!

**Comment by kdy1** [comment]
@dsherret I didn't know that I have push permission on fork. I fixed `swc_ecma_transforms` and verified that all tests pass.

**Pull request merged**